### ISSUE TITLE
Subscription feed

### ISSUE THREAD

**Issue by lennet** [uraimo/Awesome-Swift-Playgrounds#22]
Swift Playgrounds 2.0 Supports third party subscriptions https://developer.apple.com/library/content/documentation/Xcode/Conceptual/swift_playgrounds_doc_format/SubscriptionFeedFormat.html . How about creating an awesome swift playgrounds feed ?

**Comment by lennet** [comment]
yes, I used the the github releases feature for the zipping (https://github.com/lennet/image-filtering/blob/master/feed.json) and that works fine!

**Comment by macserv** [comment]
That definitely sounds like the best approach.

@lennet, do you think you could go ahead and create the JSON subscription feed now, and add those playgrounds which are currently published as a release? Even if it’s only a few items, I’m eager to subscribe.  I also feel that developers would be more likely to publish their own playgrounds as releases if the feed actually existed, in order to be included.

Once the feed is available, I’d be happy to create issues for every playground in your list asking the developers to publish a release.  Thanks for all the work you’ve done to curate this list!

**Comment by treyharris** [comment]
@lennet It looks like this didn't get followed up on, but I'm just now starting to play with the Playgrounds iOS app and it doesn't accept these URL's—including the one you link above (
https://github.com/lennet/image-filtering/blob/master/feed.json).

Checking the advertised URL's on Apple's site (which are all for Playgroundbooks requiring proprietary third-party hardware), it looks like they link to a `locales.json` URL rather than a `feed.json` URL, so perhaps something about the subscription format has changed in an undocumented way.

**Comment by lennet** [comment]
Sorry for the late response. I totally forgot about it.. Sorry the link I posted is only a link to the GitHub interface of the file and not the raw json. Adding  https://raw.githubusercontent.com/lennet/image-filtering/master/feed.json in your app should work. A localized JSON would be nicer but my content isn't localized  yet. 

Back to the original topic of the Thread:

I'm not sure how stable the subscription format is. My feed broke  often in the past due to undocumented format changes :/. I'm wondering if it makes sense to add an abstraction layer on top of  apples format
